src/wstore/asset_manager/asset_manager.py

```diff
@@ -187,7 +187,8 @@ def _load_resource_info(self, provider, data, file_=None):
         provided_as = 'FILE'
         if 'content' in data:
             if isinstance(data['content'], str) or isinstance(data['content'], unicode):
-                self._check_url_conflict(data, current_organization)
+                #We want to allow duplicate URLs as assets
```

Comment:
If the code is not used anymore, it's a better habit to remove it. If we need this feature again in the feature, we can just revert the commit.

Reply:
I have removed the code now instead of commenting